.github/actions/publish-release/README.md

```diff
@@ -0,0 +1,37 @@
+# publish-release composite action
+
+This action allows you to set up a release cycle for 
+your image without worrying about the hard stuff.
+
+Please see one of the examples in the [workflows directory](../../workflows).
+
+## Inputs
+
+### `image` (*Required*)
+
+The name of the image. This should map to `images/<name>.dockerfile` and 
+`images/<name>/` (the directory where the image resources are stored).
+
+### `dry-run` (*Optional*)
+
+Defaults to `false`. If set to anything but false, will build and tag
+the image, as well as create the slack notification, but will not
+push the release.
+
+### `github-token` (*Required*)
+
+Most of the time you can set this to `${{ secrets.GITHUB_TOKEN }}`, however 
+if you want your workflow to trigger another workflow, you should 
+supply a personal access token instead. 
```

Comment:
This is not very clear to me. As a naive reader I don't understand when or why I might have to provide a different token, or what access it would have to have.

Reply:
In fact, I don't think this documentation belongs with this action at all.  i don't have any explicit use cases in mind and it would require more docs to generate all the pieces necessary to describe why you would want to do this in this particular circumstance that i doubt anyone actually will need this.